modules/docs/lib/MDXElements.tsx

```diff
@@ -25,7 +25,7 @@ export const MDX = createComponent('div')({
  */
 const Button = (props: any) => {
   const components = useMDXComponents();
-  if (props['data-symbol'] !== undefined) {
+  if (props['data-symbol'] !== undefined && props.children) {
```

Comment:
`markdown-to-jsx` will rewrite the following:
```jsx
<button data-symbol="" className="token symbol">createStyles</button>
```

Firefox:
```jsx
<button data-symbol="" className="token symbol"></button>createStyles&gt;/button>
```

Everything else:
```jsx
<button data-symbol="" className="token symbol">createStyles</button>
```

In Firefox, that means `props.children` is `undefined` which will fail a few lines down because we access a property on `props.children`. Instead of Firefox failing to render with an error message, we simply skip it. This results in improperly formatted content in Firefox, but that is better than failing to render anything on the page.